### ISSUE TITLE
test-network-k8s: `./network kind` error creating cluster should be reported in the terminal

### ISSUE THREAD

**Issue by seanpk** [hyperledger/fabric-samples#506]
I have `docker` and `kubectl` installed as snaps, and since I'm running a VPN I needed to add a manual network using `brctl` per [docker/for-linux #1105](https://github.com/docker/for-linux/issues/1105#issuecomment-948200872).

My `./network kind` command is only getting a little further however:
```
$ ./network kind
Initializing KIND cluster "kind":
✅ - Pulling docker images for Fabric 2.3.2 ...
⚠️  - Creating cluster "kind" ...
$
```

This is pretty, but not that helpful. At the very least the user should get a message to check out the file `network-debug.log`, but publishing the error message to the console would be even better. I've attached that file ([network-debug.log](https://github.com/hyperledger/fabric-samples/files/7385684/network-debug.log)) for reference (and maybe for help with whatever my config issue is), but here are the relevant lines from it too (which look similar to my earlier VPN issue):
```
Status: Downloaded newer image for hyperledger/fabric-tools:2.3.2
docker.io/hyperledger/fabric-tools:2.3.2
Deleting cluster "kind" ...
Creating cluster "kind" ...
 • Ensuring node image (kindest/node:v1.21.1) 🖼  ...
ERROR: failed to create cluster: failed to ensure docker network: command "docker network create -d=bridge -o com.docker.network.bridge.enable_ip_masquerade=true -o com.docker.network.driver.mtu=1500 --ipv6 --subnet fc00:f853:ccd:e793::/64 kind" failed with error: exit status 1

Command Output: Error response from daemon: could not find an available, non-overlapping IPv4 address pool among the defaults to assign to the network
```

**Comment by denyeart** [comment]
@jkneubuh

**Comment by jkneubuh** [comment]
Fixed in PR #729
